### ISSUE TITLE
bug: error "/usr/bin/env: ‘node --no-warnings’: No such file or directory"

### ISSUE THREAD

**Issue by andchir** [janhq/cortex#641]
**Describe the bug**
Ubuntu 22.04.4 LTS

**To Reproduce**
1. install node 20
2. ``npm i -g @janhq/cortex``
3. ``cortex init``
~~~
/usr/bin/env: ‘node --no-warnings’: No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines
~~~

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04.4 LTS

**Comment by 0xSage** [comment]
@louis-jan potential issue/edge case we should resolve in our new installation automation 🙏

**Comment by louis-jan** [comment]
Yeah that is a known issue with nodejs `shebang`. Will figure it out, also a part of `Single Executable Node`

**Comment by louis-jan** [comment]
Fixed in #715

**Comment by louis-jan** [comment]
Addressed in 0.0.60
